app/javascript/packs/OAS3Autocomplete.js

```diff
@@ -20,22 +20,36 @@ const addAutocompleteToParam = (param: Param, accountData: AccountData): Param =
   const xDataKey = param[X_DATA_ATTRIBUTE]
   const autocompleteData = accountData[xDataKey]
   const paramHasAutocompleteData = autocompleteData && autocompleteData.length > 0 &&
-   autocompleteData.every(param => param.name !== '')
+    autocompleteData.every(param => param.name !== '')
 
   return paramHasAutocompleteData
     ? {
       ...param,
       examples: autocompleteData.reduce((examples, item) => (
-        [...examples, {summary: item.name, value: item.value}]
-      ), [{summary: X_DATA_PARAMS_DESCRIPTIONS[xDataKey], value: '-'}])
+        [...examples, { summary: item.name, value: item.value }]
+      ), [{ summary: X_DATA_PARAMS_DESCRIPTIONS[xDataKey], value: '-' }])
     }
     : param
 }
 
-const getPathParameters = (path) => {
-  const hasParamsInGet = path.get && path.get.parameters
-  return hasParamsInGet ? path.get.parameters : path.parameters
```

Comment:
How about this case when `path.get` was undefined? Is `path.parameters` still possible?

Reply:
@josemigallas @didierofrivia 
I've made some changes, it has to work in 2 scenarios:
1. When parameters is an array
Example: 
```
{
  parameters: [...]
}
```
2. When parameters comes as array, but nested in an object
Example: 
```
{
  get: {
    description: "...",
    parameters: [...]
  },
  post: {
    description: "...",
    parameters: [...]
  }
}
```